ostree/README.md

```diff
@@ -7,27 +7,24 @@ There are two ways of creating an update image:
 
 These artifacts are produced during the Yocto build.
 
-Scripts are provided for both of these, and instructions for running directly or with Docker. The Docker method has been tested running on an Ubuntu 16.04 machine only.
-
 # Creating an upgrade tarball between wic images
 
 The createOSTreeUpgrade.sh script can be used to create a field upgrade tarball.
 
 ```
-> ./createOSTreeUpgrade.sh old-wic-file new-wic-file [upgrade-tag]
+> ./createOSTreeUpgrade.sh old-wic-file new-wic-file output-file
```

Comment:
Do we need the end part of the name also?
```suggestion
> ./createOSTreeUpgrade.sh old-wic-file new-wic-file.tar.gz  output-file
```

Reply:
The build creates several files and links.
For instance:
```
console-image-lmp-imx8mmevk-20210203083927.rootfs.wic
console-image-lmp-imx8mmevk-20210203083927.rootfs.wic.gz
console-image-lmp-imx8mmevk.wic.gz
console-image-lmp-imx8mmevk.wic
```
The script can take either the `.wic `file or the `.wic.gz` file. It then either unzips them or copies them by using the `gzcat` command.
I don't think this needs to change.

Reply:
Hmm, I think I agree with @OPpuolitaival in that somewhere in the documentation it should be clarified a bit that this can take either .`*.wic` or `*.wic.tar.gz` files.

It also feels a bit confusing since the file extension isn't referenced in this line, but in L22, the files are mapped to ones with `*.wic`.

Reply:
Reworded and removed the wic extension from L22.